backend/app/db/migrations/versions/2025_02_04_1706_change_user_subscription_shema.py

```diff
@@ -0,0 +1,67 @@
+"""change user subscription shema
+
+Revision ID: ae47ad4f4179
+Revises: 3cf5570a9c36
+Create Date: 2025-02-04 17:06:31.220619
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'ae47ad4f4179'
+down_revision = '3cf5570a9c36'
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('group',
+    sa.Column('id', sa.UUID(), nullable=False),
+    sa.Column('name', sa.String(), nullable=False),
+    sa.Column('main_user_id', sa.Boolean(), nullable=False),
+    sa.Column('time_created', sa.DateTime(), nullable=False),
+    sa.Column('time_updated', sa.DateTime(), nullable=True),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table('user_subscription_history',
+    sa.Column('id', sa.Integer(), autoincrement=True, nullable=False),
+    sa.Column('user_id', sa.UUID(), nullable=False),
+    sa.Column('subscription_id', sa.UUID(), nullable=False),
+    sa.Column('group_id', sa.UUID(), nullable=False),
+    sa.ForeignKeyConstraint(['group_id'], ['group.id'], ),
+    sa.ForeignKeyConstraint(['subscription_id'], ['subscription.id'], ),
+    sa.ForeignKeyConstraint(['user_id'], ['user.id'], ),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.drop_table('user_x_subscription')
+    op.drop_column('subscription', 'expires_at')
+    op.drop_column('subscription', 'ticket_count')
+    op.add_column('user', sa.Column('ticket_count', sa.Integer(), nullable=False))

```

Comment:
![image](https://github.com/user-attachments/assets/40adede2-40f7-4117-a5ac-d34eb9ba9437)

기존 schema에서 없던 column인데 nullable=False이므로, 이전에 존재하던 Row들의 ticket_count 값이 지정되지 않아 not null constraint 에러가 발생합니다.
아래와 같이 server default 값을 지정하는게 좋을 것 같습니다! 
```suggestion
    op.add_column(
        "user",
        sa.Column("ticket_count", sa.Integer(), nullable=False, server_default="3"),
    )
```